File: lua/refactoring/tests/refactor/123/ts/single-character/inline_var.start.ts
type Order = {
    quantity: number;
    itemPrice: number;
}

// This the straight from the book (slight modifications)
function orderCalculation(order: Order) {
    const i = order.quantity*order.itemPrice;
    i;
    return i -
        Math.max(0, order.quantity - 500) * order.itemPrice * 0.05 +
        Math.min(i * 0.1, 100);
}

File: lua/refactoring/tests/refactor/123/go/single-character/inline_var.start.go
package main

import "math"

type Order struct {
    quantity  float64
    itemPrice float64
}

func orderCalculation(order Order) float64 {
    i := order.quantity * order.itemPrice
    return i - math.Max(0, order.quantity-500)*order.itemPrice*0.05 + math.Min(i*0.1, 100.0)
}
